src/Chainweb/Miner/POW.hs

```diff
@@ -114,7 +119,7 @@ powMiner logFun conf nid cutDb = runForever logFun "POW Miner" $ do
         -- Mine a new Cut
         --
         c <- _cut cutDb
-        T4 newBh payload c' adjustments' <- do
+        Ore newBh payload c' adjustments' hashAttempts <- do
```

Comment:
Could unboxed tuples be used here?

Reply:
I don't think they can be, the contents aren't primitives.

Reply:
I don't think that the content has to be primitive. The purpose of unboxed tuples is to allow functions to return more than one value without the overhead of creating a box for it.

```haskell
λ> :set -XUnboxedTuples
λ> let (# x, y #) = (# (1::Int), ("abc"::String) #)
λ> x
    1
λ> y
    "abc"
```

Reply:
The contents of the tuple are returned in registers (either as primitive values or pointers).

Reply:
https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/glasgow_exts.html?highlight=unboxed#extension-UnboxedTuples